doc/content/source/materials/InelasticRebar.md

```diff
@@ -0,0 +1,65 @@
+# Inelastic Rebar
```

Comment:
I'd suggest renaming this to PlasticRebar (because plasticity is the only form of inelasticity that it represents).

Reply:
Actually, call it PlasticTruss (to be consistent with LinearElasticTruss), which exists already.